react-accordion-component/src/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Accordion from './accordion';
+
+const arr = [
+  {
+    html: `Hypertext Markup Language(HTML) is the standard
+            markup language for creating web pages and web
+            applications.With Cascading Style Sheets(CSS) and
+            JavaScript, it forms a triad of cornerstone
+            technologies for the World Wide Web.`,
+    css: `Cascading Style Sheets(CSS) is a style sheet
+            language used for describing the presentation of a
+            document written in a markup language like HTML.
+            CSS is a cornerstone technology of the World Wide
+            Web, alongside HTML and JavaScript`,
+    js: `JavaScript, often abbreviated as JS, is a high - level,
+          interpreted programming language that conforms to
+          the ECMAScript specification JavaScript has curly -
+          bracket syntax, dynamic typing, prototype - based
+          obiect - orientation, and first - class functions.`
+  }
+]
```

Comment:
Here you should have an `array` of objects, 1 object should represent 1 item in the accordion. It should follow something similar to the following layout:

```JAVASCRIPT
const items = [
  {
    title: 'HTML',
    content: 'This is the long content about HTML',
    id: 1
  },
  {
    title: 'JavaScript',
    content: 'Here is some stuff about JS',
    id: 2
  }
];
```

You will of course need to add all the items and the correct content